pkgs/top-level/dotnet-packages.nix

```diff
@@ -169,107 +170,33 @@ let self = dotnetPackages // overrides; dotnetPackages = with self; {
     };
   };
 
-  Boogie_2_4_1 = buildDotnetPackage rec {
-    pname = "Boogie";
-    version = "2.4.1";
+  Dafny =  buildDotnetModule rec {
```

Comment:
I would prefer for this to be done in a seperate file instead of `dotnetPackages.nix`, it makes the package easier to discover IMO.


This file is mainly used for legacy `buildDotnetPackage` derivations, because of this I also believe splitting this package away would be best.